spyder/utils/tests/data/example.py

```diff
@@ -0,0 +1,63 @@
+"""
+    ********************* VerySimpleWebBrowser ************************
+
+    This is a Very Simple Web Browser implemented over Qt and QtWebKit.
+
+    author: Juan Manuel Garcia <jmg.utn@gmail.com> ~/home
+
+    *******************************************************************
+"""
```

Comment:
Sorry, don't do it. I saw you're testing that `get_words` gets words in this string.